tests/js/spec/bootstrap.spec.jsx

```diff
@@ -0,0 +1,33 @@
+import * as Sentry from '@sentry/browser';
+
+describe('bootstrap', function() {
+  it('configures Sentry', async function() {
+    window.__initialData = {
+      distPrefix: 'dist',
+      csrfCookieName: 'csrf',
+      userIdentity: {
+        id: 1,
+        email: 'foo@example.com',
+      },
+      sentryConfig: {
+        dsn: 'https://public@example.com/1',
+        environment: 'development',
+        release: '123',
+      },
+    };
+
+    require('app');
+
+    const eventId = await Sentry.captureMessage('test');
+    expect(eventId).toBeDefined();
+    expect(Sentry.testkit.reports()).toHaveLength(1);
+    const {originalReport} = Sentry.testkit.reports()[0];
```

Comment:
had to do this as sentry-testkit coerces the json payload into a 'report', but that report is missing some attributes (like release)

Reply:
@dcramer it is true we're omitting some attributes. Specifically, `release`,`user` we do pass over:
https://github.com/wix/sentry-testkit/blob/master/src/index.js#L29
I do agree though that `environment` should be part of  `SentryTestkit.Report`